api/scripts/server.ts

```diff
@@ -0,0 +1,19 @@
+import fastify from 'fastify';
+
+const server = fastify();
+
+server.all('/*', async () => {
+  // TODO: Build a fake APIGatewayEvent and Context? object to pass into the handler
+  // const event: APIGatewayEvent = {};
+  // const context: Context = {};
+  // return handler(event, context);
+  return 'Hello World';
```

Comment:
I feel like this PR should be set up to forward to our lambda so we can also test the dummy endpoint you created.

Reply:
I figured we could do that once we know how to build the fake `event` and `context` parameters. All we'd have to do is uncomment the code above and do that, so we can do that in a future PR